### ISSUE TITLE
ITT during take off

### ISSUE THREAD

**Issue by kutak11** [guifarias31/msfs_tbm930_project#15]
Hi,
I know there are known issues about ITT. But is it really normal that during take off roll after torque setting about 95%-100% the ITT is flashing red with master warning lights? (overheating) Or I am doing something wrong?

Thank you
Tomas K.

**Comment by guifarias31** [comment]
Yeah, it's that way on the vanilla plane as well. As I mentioned on the "know issues" section, if I decrease the ITT it stays too low on cruise, and if I increase it, it stays to high on takeoff / on the ground.

**Comment by withinboredom** [comment]
I have an idea of how to fix this. I'll give it a go sometime later this week.

**Comment by withinboredom** [comment]
I’ve found the exact variable that’s incorrect but don’t know what the exact value is supposed to be. Does anyone know where I can find a recording of the MFD during takeoff/cruise/taxi of a real plane (along with what the ISA is)? The POH isn’t helpful here.

**Comment by JokeyChen** [comment]
@withinboredom I have a copy of HotStart TBM 900 from XPlane which has a very accurate model. Although it's not TBM 930, I think for our purposes it should be close enough. I can run a flight tonight or tomorrow to record the ITT during the whole flight if you want.

**Comment by kutak11** [comment]
I think for this purpose it will be accurate enough.
Thank you

**Comment by JokeyChen** [comment]
Here's some data I recorded from my recent flights.

I forgot to record the data during taxi and takeoff, so those two data are from another flight. That's why the ISA temp is quite different from the other recorded data.

Phases | ISA Temp | TRQ | ITT
------------ | ------------- | ----------- | ----------- 
Taxi | +12 C | - | 642 C
Takeoff | +10 C | 99 | 775 C
10000 ft | -1 C | 92 | 706 C
20000 ft | 1 C | 98 | 789 C
25000 ft | -1 C | 97 | 832 C
30000 ft | -1 C | 83 | 837 C
Cruise @ 31000 ft | 0 C | 87 | 839 C

For descending, the ITT varies based on the torque settings I set so I don't think that values too much here.

**Comment by kutak11** [comment]
nice data... @withinboredom are you able to fix it according this? In my opinion there is not bigger issue than this at this moment.

**Comment by withinboredom** [comment]
Oh, that's some nice data. I'll try and jump on tonight/tomorrow. I've been pretty swamped with work this week, but it looks like things will slow down soon.

**Comment by kutak11** [comment]
hi.. how is it doing with ITT warning issue? are these data sufficient?

**Comment by withinboredom** [comment]
Yeah, these numbers will work. I've been pretty busy with work lately and I was sick the last couple of days. Will hopefully get to it soon.

**Comment by withinboredom** [comment]
@JokeyChen I'd love it if you could apply the changes in #25 and see how that flies. You can also try setting `itt_tuning_constant` to `0.62` as that "felt" more realistic, but `0.66` lined up with your numbers the best.

**Comment by withinboredom** [comment]
It's been released in 0.5.3, please let me know how it goes!

**Comment by JokeyChen** [comment]
@withinboredom Thank you for your effort! I'll give that a try in a day or two.

**Comment by kutak11** [comment]
Hi, I still have ITT overheating during take off... or am I doing anything wrong? Power set about 95%

**Comment by Andydigital** [comment]
> Hi, I still have ITT overheating during take off... or am I doing anything wrong? Power set about 95%

What is the ambient temperature and altitude at the location you have the issue? Or is it everywhere?

**Comment by kutak11** [comment]
everywhere.. now I tried 3 takeoffs and all with ITT warning.. runways at about 500-1000ft, 15°C. To be honest its worse than before. Red alert is flashing a bit longer

**Comment by JokeyChen** [comment]
I can confirm that the issues have become worse. At KTEX @ ISA temp, I only get around 50% torque if I want to keep ITT in the green.

**Comment by e-racer** [comment]
> I can confirm that the issues have become worse. At KTEX @ ISA temp, I only get around 50% torque if I want to keep ITT in the green.

I can also confirm that.. departed 3 times near sealevel Runways (LICT, LFKJ, LIRJ) and around 12-16degrees OAT .. but anyway many thx for your work until now!

**Comment by withinboredom** [comment]
Thanks for your invaluable feedback everyone! 

> Hi, I still have ITT overheating during take off... or am I doing anything wrong? Power set about 95%

From my understanding of watching many youtube videos, you don't actually stay at high % TRQ after you have wheels up and until you turn off the inertial separator, you're ITT limited. That being said, how much TRQ you can actually get depends heavily on weather conditions and altitude about MSL.

> I can confirm that the issues have become worse.

We're working on it!

**Comment by inztigator** [comment]
This might help...

https://pohperformance.com/TBM/index.html

Provides you with the torque based on real calculations and a table for reference.

**Comment by JokeyChen** [comment]
@withinboredom No worries! I didn't meant to pressure anyone. Just want to offer some data points.

@inztigator I've been using this app (both for TBM in XPlane and for C172 & C182 for my real world flying) and this app is simply amazing. I still can't believe to this date that it is free. But I don't think it would help too much on the ITT calculations.

**Comment by inztigator** [comment]
I think it could be useful. It has the Torque table and you can adjust the temperature and it will give you the values...

**Comment by kutak11** [comment]
I have just notice that ITT has some conectivity with brakes. If I stay on brakes just a bit longer before take off roll the ITT shows overheating... when released temprature drops down quickly

**Comment by guifarias31** [comment]
> everywhere.. now I tried 3 takeoffs and all with ITT warning.. runways at about 500-1000ft, 15°C. To be honest its worse than before. Red alert is flashing a bit longer

Decided to revert back to default ITT values for now.
